bindings/d/package.d

```diff
@@ -0,0 +1,3608 @@
+	discardColour0    = discardColor0,
+	discardColor1     = 0x0010, ///Discard frame buffer attachment 1.
+	discardColour1    = discardColor1,
+	discardColor2     = 0x0020, ///Discard frame buffer attachment 2.
+	discardColour2    = discardColor2,
+	discardColor3     = 0x0040, ///Discard frame buffer attachment 3.
+	discardColour3    = discardColor3,
+	discardColor4     = 0x0080, ///Discard frame buffer attachment 4.
+	discardColour4    = discardColor4,
+	discardColor5     = 0x0100, ///Discard frame buffer attachment 5.
+	discardColour5    = discardColor5,
+	discardColor6     = 0x0200, ///Discard frame buffer attachment 6.
+	discardColour6    = discardColor6,
+	discardColor7     = 0x0400, ///Discard frame buffer attachment 7.
+	discardColour7    = discardColor7,
+	discardDepth      = 0x0800, ///Discard frame buffer depth attachment.
+	discardStencil    = 0x1000, ///Discard frame buffer stencil attachment.
+	discardColorMask  = 0x07F8,
+	discardColourMask = discardColorMask,
+	discardMask       = 0x1FF8,
+}
+
+/**
+* Rendering state discard. When state is preserved in submit, rendering states can be discarded
+* on a finer grain.
+*/
+alias Discard_ = ubyte;
+enum Discard: Discard_{
+	none           = 0x00, ///Preserve everything.
+	bindings       = 0x01, ///Discard texture sampler and buffer bindings.
+	indexBuffer    = 0x02, ///Discard index buffer.
+	instanceData   = 0x04, ///Discard instance data.
+	state          = 0x08, ///Discard state and uniform bindings.
+	transform      = 0x10, ///Discard transform.
+	vertexStreams  = 0x20, ///Discard vertex streams.
+	all            = 0xFF, ///Discard all states.
+}
+
+alias Debug_ = uint;
+enum Debug: Debug_{
+	none       = 0x0000_0000, ///No debug.
+	wireframe  = 0x0000_0001, ///Enable wireframe for all primitives.
+/**
+Enable infinitely fast hardware test. No draw calls will be submitted to driver.
+It's useful when profiling to quickly assess bottleneck between CPU and GPU.
+*/
+	ifh        = 0x0000_0002,
+	stats      = 0x0000_0004, ///Enable statistics display.
+	text       = 0x0000_0008, ///Enable debug text display.
+	profiler   = 0x0000_0010, ///Enable profiler. This causes per-view statistics to be collected, available through `bgfx::Stats::ViewStats`. This is unrelated to the profiler functions in `bgfx::CallbackI`.
+}
+
+alias BufferComputeFormat_ = ushort;
+enum BufferComputeFormat: BufferComputeFormat_{
+	_8x1   = 0x0001, ///1 8-bit value
+	_8x2   = 0x0002, ///2 8-bit values
+	_8x4   = 0x0003, ///4 8-bit values
+	_16x1  = 0x0004, ///1 16-bit value
+	_16x2  = 0x0005, ///2 16-bit values
+	_16x4  = 0x0006, ///4 16-bit values
+	_32x1  = 0x0007, ///1 32-bit value
+	_32x2  = 0x0008, ///2 32-bit values
+	_32x4  = 0x0009, ///4 32-bit values
+	shift  = 0,
+	mask   = 0x000F,
+}
+
+alias BufferComputeType_ = ushort;
+enum BufferComputeType: BufferComputeType_{
+	int_   = 0x0010, ///Type `int`.
+	uint_  = 0x0020, ///Type `uint`.
+	float_ = 0x0030, ///Type `float`.
```

Comment:
If you capitalized first letter, you wouldn't need to add this unnatural `_` at the end.

Reply:
> If you capitalized first letter, you wouldn't need to add this unnatural `_` at the end.

This complies with the D Style, and is typical across the standard library. A D user would find this pretty normal. Capitalising enum members, however, is completely unnatural because they look like nested types.